docs/requirements.md

```diff
@@ -1,51 +1,143 @@
+#### Analytics
+{:.no_toc}
+- Not applicable. Cosmos DB always returns committed records, so there are no Cosmos DB–specific requirements.
 
-## JDBC database recommendations
+</div>
 
-In ScalarDB on JDBC databases, you can't choose a consistency level (`LINEARIZABLE`, `SEQUENTIAL` or `EVENTUAL`) in your code by using the `Operation.withConsistency()` method. In addition, the consistency level depends on the setup of your JDBC database.
+<div id="Cassandra" class="tabcontent" markdown="1">
 
-For example, if you have asynchronous read replicas in your setup and perform read operations against them, the consistency will be eventual because you can read stale data from the read replicas. On the other hand, if you perform all operations against a single master instance, the consistency will be linearizable.
+#### Transactions
+{:.no_toc}
+- Use a single primary cluster for all operations (no read or write operations in non-primary clusters).
+- Use `batch` or `group` for `commitlog_sync`.
+- If you're using Cassandra-compatible databases, those databases must properly support lightweight transactions (LWT).
 
-With this in mind, you must perform all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. In other words, ScalarDB does not support read replicas. 
+#### Analytics
+{:.no_toc}
+- Not applicable. Cassandra always returns committed records, so there are no Cassandra-specific requirements.
+
+</div>
+</div>
+
+## Recommendations
+
+Properly configuring each underlying database of ScalarDB for high performance and high availability is recommended. The following recommendations include some knobs and configurations to update.
 
 {% capture notice--info %}
 **Note**
 
-You can still use a read replica as a backup and standby even when following this guideline.
+ScalarDB can be seen as an application of underlying databases, so you may want to try updating other knobs and configurations that are commonly used to improve efficiency.
 {% endcapture %}
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+<div id="tabset-2">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'JDBC_databases2', 'tabset-2')" id="defaultOpen-2">JDBC databases</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB2', 'tabset-2')">DynamoDB</button>
+  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL2', 'tabset-2')">Cosmos DB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'Cassandra2', 'tabset-2')">Cassandra</button>
+</div>
+
+<div id="JDBC_databases2" class="tabcontent" markdown="1">
+- Use read-committed isolation for better performance.
```

Comment:
Just a confirmation. Are you intentionally not mentioning the JDBC transaction manager since it's not officially explained in the docs? If so, it's fine, but I just thought the requirement and recommendation would be application-dependent when using the JDBC transaction manager.

Also, I would recommend users configure the isolation level (read-committed) at the underlying database setting instead of setting by `scalar.db.jdbc.isolation_level` for performance reasons since it will set the isolation level every time getting connections and reduce performance. But if too specific and detailed, forget about it.

Reply:
> Just a confirmation. Are you intentionally not mentioning the JDBC transaction manager since it's not officially explained in the docs? If so, it's fine, but I just thought the requirement and recommendation would be application-dependent when using the JDBC transaction manager.

I think this is a good point. We have the JDBC transaction manager, and we finally decided not to delete it. So maybe we should mention that this document is only for the Consensus Commit transaction manager? (Also, we should revert the JDBC transaction manager things in the document)

Reply:
@jnmt Thanks! It's a good point.
Since the JDBC transaction manager is an expert option and we don't expect normal users to use it, I didn't mention it.
Maybe, I can add a note for that, just in case.

@brfrn169 What do you think?